### PR TITLE
Add SubmitPPM Handler

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Add handler to change PPM's status to SUBMITTED for HHG_PPM flow

## Reviewer Notes
- Slowly separating the submission logic for PPMs, HHGs and moves. Currently the submission of an HHG is tied to also submitting a PPM and submitting a move. This handler will only submit a PPM to avoid any funky interactions that relied on the move being submitted

## Setup
`make server_run`
`make server_test`

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/161972310) for this change

### CODE REVIEW

File: pkg/handlers/internalapi/personally_procured_move.go [hunk 1]
```diff
@@ -255,6 +255,40 @@ func (h PatchPersonallyProcuredMoveHandler) Handle(params ppmop.PatchPersonallyP
 	return ppmop.NewPatchPersonallyProcuredMoveOK().WithPayload(ppmPayload)
 }
 
+// SubmitPersonallyProcuredMoveHandler Submits a PPM
```
Comment (kahlouie) [inline pkg/handlers/internalapi/personally_procured_move.go]: Can we move all this new code to below the next function which uses the `PatchPersonallyProcuredMoveHandler` struct definition on line 211 to keep related code closer together?

File: swagger/internal.yaml [hunk 2]
```diff
@@ -3528,6 +3557,7 @@ paths:
           description: no files to be processed into attachments PDF
         500:
           description: server error
+
```
Comment (kahlouie) [inline swagger/internal.yaml]: Nit pick, but should this empty line be here?

File: pkg/handlers/internalapi/personally_procured_move.go (hunk 1 above)
Comment (Ronolibert) [reply pkg/handlers/internalapi/personally_procured_move.go]: Will do!

File: swagger/internal.yaml (hunk 2 above)
Comment (Ronolibert) [reply swagger/internal.yaml]: I think prettier added on the pre-commit

**Pull request merged**